can_tools/scrapers/official/federal/HHS/facility.py

```diff
@@ -0,0 +1,151 @@
+
+    def fetch(self):
+        return self.dataset_details()
+
+    def normalize(self, data: str) -> pd.DataFrame:
+        # Read the dataframe from the string csv
+        df = pd.read_csv(StringIO(data))
+        df.columns = [x.lower().strip() for x in df.columns]
+
+        # Set date and fips code
+        df.loc[:, "dt"] = pd.to_datetime(df["collection_week"])
+
+        # Filter out all of the columns without a fips code for now -- I
+        # think that it is likely that we could reverse engineer these
+        # either by looking them up or by mapping city to county
+        df = df.loc[~df["fips_code"].isna(), :]
+        # :see_no_evil:
+        df["location"] = (
+            df["fips_code"]
+            .astype(int)
+            .replace(
+                {
+                    # 02120 corresponded to Kenai-Cook Inlet Division... It was
+                    # then the relevant piece became Kenai Peninsula Borough which
+                    # is 02122
+                    # https://data.nber.org/asg/ASG_release/County_City/FIPS/FIPS_Changes.pdf
+                    2120: 2122,
+                    # City associated with the hospital is Seward which is in the
+                    # Kenai Borough which is 02122 but I have no idea how this
+                    # ended up with fips code 02210???
+                    # https://en.wikipedia.org/wiki/Seward,_Alaska
+                    2210: 2122,
+                    # 02260 was fips code for Valdez-Chitina-Whittier Division... It
+                    # was then put into Valdez–Cordova Census Area which is
+                    # 02261, but 02261 was split in Jan 2019 and we'll need to change
+                    # this again if we update geographies
+                    # https://data.nber.org/asg/ASG_release/County_City/FIPS/FIPS_Changes.pdf
+                    2260: 2261,
+                    # 02280 corresponded to Wrangell-Petersburg but became the
+                    # Petersburg Borough 02195 in 2012
+                    # https://www.cdc.gov/nchs/nvss/bridged_race/county_geography-_changes2015.pdf
+                    2280: 2195,
+                    # City associated with the hospital is Cordova which is in the
+                    # Valdez-Cordova census area but I don't know which one this
+                    # ended up in after the split...
+                    # https://en.wikipedia.org/wiki/Cordova,_Alaska
+                    2080: 2261,
+                }
+            )
+        )
+
+        # Set all missing values (-999999) to nan for all numeric columns
+        numeric_cols = list(df.select_dtypes("number"))
+        df.loc[:, numeric_cols] = df.loc[:, numeric_cols].where(lambda x: x > 0, np.nan)
+
+        # Create new columns that we need
+        df["inpatient_beds_used_covid_7_day_avg"] = df.eval(
+            "total_adult_patients_hospitalized_confirmed_covid_7_day_avg + "
+            "total_pediatric_patients_hospitalized_confirmed_covid_7_day_avg"
```

Comment:
Do other source  include pediatric in "inpatient beds"?

I have a hunch that inpatient is adult and pediatric, if counted, would be reported separately

Reply:
This is an important question.

I'm going to vote for using the sum because I think they're using the sum of pediatric and adult beds in other places (they report adult inpatient beds in a separate variable).

Here's the data dictionary: https://healthdata.gov/covid-19-reported-patient-impact-and-hospital-capacity-facility-data-dictionary